composer.json

```diff
@@ -76,7 +76,7 @@
         }
     ],
     "require-dev": {
-        "appwrite/sdk-generator": "0.19.5",
+        "appwrite/sdk-generator": "0.18.8",
```

Comment:
note to self, put sdk-generator back up to normal version once console SDK patch is implemented